docs/sources/whatsnew/whats-new-in-v9-1.md

```diff
@@ -0,0 +1,188 @@
+---
+_build:
+  list: false
+aliases:
+  - /docs/grafana/latest/guides/whats-new-in-v9-1/
+description: Feature and improvement highlights for Grafana v9.1
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '9.1'
+  - release notes
+title: What's new in Grafana v9.1
+weight: -33
+---
+
+# What's new in Grafana v9.1
+
+We're excited to announce Grafana v9.1, with a variety of improvements that focus on Grafana's usability, performance, and security. Read on to learn about new options to share and embed dashboards, search and navigation enhancements, new panel options and additional authentication features. You can also find out more about new Single Sign-On and role-based access control options in Grafana Enterprise, and more. For details, see the complete [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```

Comment:
Are you using the line break best practice where one sentence is on each line? Is that something you want me to check for? 
```suggestion
We're excited to announce Grafana v9.1, with a variety of improvements that focus on Grafana's usability, performance, and security. 
Read on to learn about new options to share and embed dashboards, search and navigation enhancements, new panel options and additional authentication features. 
You can also find out more about new Single Sign-On and role-based access control options in Grafana Enterprise, and more. 
For details, see the complete [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```